.travis.yml

```diff
@@ -7,6 +7,7 @@ services:
   - mongodb
 
 install:
+  - echo y | pecl install mongo
```

Comment:
Pasting in extracts of the GItter chat for future reference.

> two mongo extensions for php now
https://pecl.php.net/package/mongo
https://pecl.php.net/package/mongodb